webpack/components/Table/EmptyStateMessage.js

```diff
@@ -55,11 +77,47 @@ const EmptyStateMessage = ({
         <EmptyStateBody>
           {emptyStateBody}
         </EmptyStateBody>
+        {showPrimaryAction &&
+          <Button>
+            <a href={primaryActionLink} style={{ color: 'white', textDecoration: 'none' }}>{primaryActionTitle}</a>
+          </Button>}
+        {showSecondaryAction &&
+          <EmptyStateSecondaryActions>
+            <Button variant="link">
+              <a href={secondaryActionLink} style={{ textDecoration: 'none' }}>{secondaryActionTitle}</a>
+            </Button>
+          </EmptyStateSecondaryActions>
+        }
+
+        {searchIsActive && !filtersAreActive &&
+          <EmptyStateSecondaryActions>
+            <Button variant="link" onClick={clearSearch}>
+              Clear search
+            </Button>
+          </EmptyStateSecondaryActions>}
+
+        { !!filtersAreActive && !searchIsActive &&
+          <EmptyStateSecondaryActions>
+            <Button variant="link" onClick={clearFilters}>
+              Clear filters
+            </Button>
+          </EmptyStateSecondaryActions>
+        }
+
+        {searchIsActive && !!filtersAreActive &&
+          <EmptyStateSecondaryActions>
+            <Button variant="link" onClick={clearFiltersAndSearch}>
+              Clear search
+            </Button>
+          </EmptyStateSecondaryActions>
+        }
```

Comment:
This section could be simplified with something like
```js
const secondaryActionText = filtersAreActive ? __('Clear filters') : __('Clear search')
```
Then below, you can just write
```js
{secondaryActionText}
```

Reply:
I think it can be simplified even more, so you don't have to repeat the `<EmptyStateSecondaryActions />`, if you write a `handleClick` function that looks at `filtersAreActive` and `searchIsActive`.

Reply:
Ok, gotcha.